src/helm/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Changelog
+
+All notable changes to this helm chart will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+## [0.2.0] - 2023-11-08
```

Comment:
Do you think you can manually add tags to the repository pointing to those changes, and add related link at the end of this changelog?

Reply:
Done!